### DISCUSSION TITLE
🔀 OpenSSL 3.0.8 7 Feb 2023 not supported

### DISCUSSION THREAD

**Discussion by qwertmax** [isontheline/pro.webssh.net#880] [Q&A]
Is it possible to add support of `OpenSSH_8.7p1, OpenSSL 3.0.8 7 Feb 2023`

It looks like app is not supported it, because it keeps asking password when I provided .pem key. 

P.S. .pem key working perfectly for `OpenSSH_7.9p1 Debian-10+deb10u2, OpenSSL 1.1.1n  15 Mar 2022`

**Comment by isontheline** [comment]
Moved to #881
